modules/admin_manual/pages/configuration/server/security/hsmdaemon/index.adoc

```diff
@@ -403,21 +382,20 @@ You can do so, using the following command (which also shows example console out
 
 === Configure ownCloud
 
-[TIP]
+[IMPORTANT]
 ====
-If anyone accesses ownCloud while encryption is enabled, it will automatically generate the keys. 
-To prevent this, shut down the web server until encryption is appropriately configured.
+If anyone accesses ownCloud while encryption is enabled, it will automatically generate the keys. To prevent this, shut down the web server until encryption is appropriately configured.
```

Comment:
```suggestion
If anyone accesses ownCloud while encryption is enabled, it will automatically generate the keys. To prevent this shut down the web server until encryption is appropriately configured.
```